src/Validation/Validation.php

```diff
@@ -1021,6 +1023,75 @@ public static function uploadedFile($file, array $options = [])
     }
 
     /**
+     * Validates a geographic coordinate.
+     *
+     * Supported formats are right now:
+     *
+     * - <latitude>, <longitude> Example: -25.274398, 133.775136
+     *
+     * ### Options
+     *
+     * - `type` - A string of the coordinate format, right now only `longLat`.
+     * - `latLong` - By default `both`, can be `long` and `lat` as well to validate
+     *   only a part of the coordinate.
+     *
+     * @param string $value Geographic location as string
+     * @param array $options Options for the validation logic.
+     * @return bool
+     */
+    public static function geoCoordinate($value, array $options = [])
+    {
+        $options += [
+            'latLong' => 'both',
+            'type' => 'latLong'
+        ];
+        if ($options['type'] === 'latLong') {
+            if ($options['latLong'] === 'both') {
+                $pattern = '/^' . self::$_pattern['latitude'] . ',\s*' . self::$_pattern['longitude'] . '$/';
+            }
+            if ($options['latLong'] === 'long') {
+                $pattern = '/^' . self::$_pattern['longitude'] . '$/';
+            }
+            if ($options['latLong'] === 'lat') {
+                $pattern = '/^' . self::$_pattern['latitude'] . '$/';
+            }
+        } else {
+            throw new \RuntimeException(sprintf('Unsupported coordinate type "%s".', $options['type']));
+        }
+        return (bool)preg_match($pattern, $value);
```

Comment:
What happens with float values when someone has done something silly like `setlocale('de_DE')`. From what I remember that makes string values of floats change their decimal separator.